README.md

```diff
@@ -49,14 +49,17 @@ at the end of its full name, e.g., `"foo.Bar$"`.
 Available operations are:
 
 * `reflectClassByName()`: enables `Reflect.getClassForName`
-* `reflectDeclaredConstructors()`: enables `Reflect.getDeclaredConstructors`
+* `reflectEnumerateClass()` enables `Reflect.enumerateClasses`
+* `reflectDeclaredConstructors()`: enables `Reflect.getDeclaredConstructors` and `Reflect.getAllClasses`
 * `reflectModuleAccessor()`: enables `Reflect.loadModule`
 
 ## API
 
 * `be.doeraene.sjsreflect.Reflect`
   * `Reflect.getClassForName(fqcn: String): Class[_]`:
     find a class by its name
+* `Reflect.enumerateClasses: js.Array[Class[_]]`:
+    enumerate all classes specified by the `reflectEnumerateClass()` operator (and descendants if the  `selectDescendentClasses` selector is used)
```

Comment:
operator -> operation (as used elsewhere in the readme)

The part "(and descendants if the `selectDescendentClasses` selector is used)" is redundant. It is a consequence of how selectors work. There is no reason to add this here but not in other operations; and we definitely don't want to repeat it on every operation. So it should be removed.